esp/src/src-react/components/DFSExists.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService, Result } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+const logger = scopedLogger("src-react/components/DFSExist.tsx");
+
+const myDaliService = new DaliService({ baseUrl: "" });
+
+interface DFSExistProps {
+}
+
+export const DFSExists: React.FunctionComponent<DFSExistProps> = ({
+
+}) => {
+
+    const [columns, setColumns] = React.useState<IColumn[]>([]);
+    const [items, setItems] = React.useState<any[]>([]);
+    const [fileName, setFileName] = React.useState<string>("");
+
+    const onSubmit = React.useCallback(() => {
+        myDaliService.DFSExists({ FileName: fileName }).then(response => {
+            const data = csvParse(response.Result);
```

Comment:
I didn't look at the API earlier, but the npm build step is failing now because `response` in this case is a boolean and not an object, and so doesn't have a `Result`. Also, since it just returns a true or false, this callback would be different than the other components - there won't be any data to set, and the HTML of the component would be different as well. I'm not sure how though.